packages/ethers-5/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "@web3modal/ethers-5",
+  "version": "3.1.0",
+  "type": "module",
+  "main": "./dist/esm/exports/index.js",
+  "types": "./dist/types/exports/index.d.ts",
+  "files": [
+    "dist",
+    "!tsconfig.tsbuildinfo"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/types/exports/index.d.ts",
+      "import": "./dist/esm/exports/index.js",
+      "default": "./dist/esm/exports/index.js"
+    },
+    "./react": {
+      "types": "./dist/types/exports/react.d.ts",
+      "import": "./dist/esm/exports/react.js",
+      "default": "./dist/esm/exports/react.js"
+    },
+    "./vue": {
+      "types": "./dist/types/exports/vue.d.ts",
+      "import": "./dist/esm/exports/vue.js",
+      "default": "./dist/esm/exports/vue.js"
+    }
+  },
+  "typesVersions": {
+    "*": {
+      "react": [
+        "./dist/types/exports/react.d.ts"
+      ],
+      "vue": [
+        "./dist/types/exports/vue.d.ts"
+      ]
+    }
+  },
+  "scripts": {
+    "build:clean": "rm -rf dist",
+    "build": "npm run build:clean; tsc --build",
+    "watch": "npm run build:clean; tsc --watch",
+    "typecheck": "tsc --noEmit",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+  },
+  "dependencies": {
+    "@coinbase/wallet-sdk": "3.7.2",
+    "@walletconnect/ethereum-provider": "2.10.1",
+    "@web3modal/polyfills": "3.1.0",
+    "@web3modal/scaffold": "3.1.0",
+    "@web3modal/scaffold-react": "3.1.0",
+    "@web3modal/scaffold-vue": "3.1.0",
+    "@web3modal/utils": "3.1.0"
+  },
+  "peerDependencies": {
+    "ethers": "5.7.2",
```

Comment:
This restricts whole package to just one version of ethers, make it `>=5` and less than or equal to 6 (don't remember the syntax from the top of my head though).